libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -214,10 +214,13 @@ class AppHeaderCore extends Component<IAppHeaderProps & WrappedComponentProps, I
     });
 
     private getTrialCountdown = (expiredDate: string) => {
-        const trialDaysLeft = differenceInCalendarDays(new Date(expiredDate), new Date()) + 1; // include expired date.
-        if (trialDaysLeft === 1) {
+        const trialDaysLeft = differenceInCalendarDays(new Date(expiredDate), new Date()) - 1; // subtract expired date
+        if (trialDaysLeft === 0) {
             return <FormattedMessage id="gs.header.countdown.lastDay" />;
         }
+        if (trialDaysLeft === 1) {
+            return <FormattedMessage id="gs.header.countdown.oneDayLeft" />;
```

Comment:
As mentioned in the other comment, maybe we don't have to define a new translationId `"gs.header.countdown.onDayLeft"`. Please delete lines 221 ~ 223 instead.